docs/stencil-docs/storefront-customization/custom-templates.md

```diff
@@ -91,9 +91,7 @@ The requirements and options for the `.stencil` or `config.stencil.json` (if usi
 
 * The HTML files must reside in either the brand, category, product, or page subdirectories.
 
-* All brand URLs are nested under the /brands/ parent. Use URL encoding with brand URLs.
-
-* That parent for brand URLs is /brands/ (plural), while the corresponding subdirectory for HTML files is /brand/ (singular).
+* Use URL encoding for special characters in URLs.
```

Comment:
consider adding a conditional clause at the beginning of this point, something like 

```suggestion
When [verbing] [brand category product page] URLs, [make sure to / you must] convert any special characters to their [URL-encoded equivalents](link to external ref on URL-encoding, maybe mdn).
```